src/Containers/SearchFormContainer/SearchFormContainer.jsx

```diff
@@ -0,0 +1,22 @@
+import { AddMovie } from '../AddMovie/AddMovie';
+import { SearchForm } from '../../Components/SearchForm/SearchForm';
+import { useSearchParams } from 'react-router-dom';
+
+export const SearchFormContainer = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
```

Comment:
This can also be moved inside `SearchForm` itself to encapsulate all this logic there.